charts/argoconfig/Chart.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: v2
 name: argoconfig
 description: Configure Argo CD AppProjects and Applications
 type: library
-version: 0.7.4
+version: 0.7.5
```

Comment:
Do we have to update the dependency to the `argoconfig` chart in the `*-apps` or are we just going to ignore it, as it is not actually an upgrade?

Reply:
we need to publish the new versions before we can update dependants

this change only really touches docs, so bumping anything that depends on `argoconfig` and `common` doesn't need a high priority as it is indeed a cosmetic change, i'm assuming we'll do it anyway nevertheless

Reply:
Right, we can't bump until the new version is actually being published. Regarding the priority of bumping I'm totally with you.